hack/update-installer.sh

```diff
@@ -60,9 +69,9 @@ do
     fi
 done
 
-for file_name in $(ls $DEPLOY_PATH)
```

Comment:
can't we just recursively iterate over the config dir files?

Reply:
there are YAML files that we don't want to use there

Reply:
so could we recurse only the dirs we want to use?

Reply:
there is a ticket on this file, I will handle all the problems in this file in the ticket